index.bs

```diff
@@ -5444,18 +5447,14 @@ ECMAScript value or failure, or the steps may throw an exception.
       <dd>
         1. If Type(|value|) is not Object, return failure.
 
-        2. Let |hop| be HasOwnProperty(|value|, |identifier|)
-
-        3. Assert: |hop| is not an abrupt completion.
+        2. Let |hop| be [=!=] HasOwnProperty(|value|, |identifier|)
 
-        4. If |hop| is false, return failure.
+        3. If |hop| is false, return failure.
 
-        5. Let |value| be |value|.[<span>[</span>Get]](|identifier|,
+        4. Let |value| be [=!=] |value|.[<span>[</span>Get]](|identifier|,
```

Comment:
Same question, but here you'd need to prevent `value` from having any getters and having anything with a potential getter in its prototype chain.